feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-product-images/product-image-zoom-product-images.component.html

```diff
@@ -1,5 +1,15 @@
 <ng-container *ngIf="mainImage$ | async as main">
-  <cx-media [container]="main" (click)="triggerZoom(true)"></cx-media>
+  <cx-media
+    *cxFeature="'useExtendedMediaComponentConfiguration'"
+    [container]="main"
+    (click)="triggerZoom(true)"
+    [elementType]="'img'"
+  ></cx-media>
+  <cx-media
+    *cxFeature="'!useExtendedMediaComponentConfiguration'"
+    [container]="main"
+    (click)="triggerZoom(true)"
+  ></cx-media>
```

Comment:
query: this usage of cx-media doesn't contain format property. Is the feature toggle wrapper required?